yarn-project/ethereum/src/deploy_l1_contracts.ts

```diff
@@ -375,7 +375,7 @@ export const deployL1Contracts = async (
     feeAssetAddress.toString(),
     args.l2FeeJuiceAddress.toString(),
   ]);
-  logger.info(`Deployed Fee Juice Portal at ${feeJuicePortalAddress}`);
+  logger.verbose(`Deployed Fee Juice Portal at ${feeJuicePortalAddress}`);
```

Comment:
Just checking whether this is on L1? If so, I believe current consensus is to name that "Fee Asset".

Reply:
Fun fact: "fee juice" (or feeJuice, or fee-juice, etc) appears exactly 500 times on our codebase.

Reply:
Sorry, that was just across typescript. It's actually 796 times.

Reply:
Will be a fun PR to try and separate fee juice from fee asset in the code. 

Also I meant to comment on line 325 🤦🏼‍♂️